modules/ossm-operator-install.adoc

```diff
@@ -83,3 +83,74 @@ $ oc new-project istio-system
 $ oc apply -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.12/deploy/servicemesh-operator.yaml
 ----
 
+= Installing the Operators using OperatorHub Web UI
+
+Jaeger and Kiali operators must be installed before the {ProductShortName} operator. 
+
+== Installing the Jaeger Operator
+You must install the Jaeger Operator for the {ProductName} Operator to install the control plane.
+
+.Prerequisites
```

Comment:
Isn't installing Elasticsearch a prerequisite for the Jaeger installation?

Reply:
Does't look this way, checked Jaeger operator sources, they install Elasticsearch themselves (and do not declare a dependency on it).

Reply:
It's probably not technically a dependency because you only need Elasticsearch when you install the "production" strategy of Jaeger.  But if you're deploying Jaeger in production, you need a data store and the Maistra README documents installing Elasticsearch.  https://github.com/Maistra/istio-operator  I wasn't sure if the operators had changed or not, so thought it was better to ask than have a missing step in the installation.